__tests__/api/topics/follow.test.js

```diff
@@ -69,7 +71,7 @@ describe('PUT /topics/follow', () => {
     // Assertion
     expect(response.statusCode).toBe(200);
     expect(response.body.message).toBe('Success delete topic user');
-  });
+  }); */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Continuation of the previous comment. The closing comment tag should also be removed.
```diff
-  }); */
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->